indico/modules/events/contributions/util.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+from sqlalchemy.orm import load_only, contains_eager
+
+from indico.modules.events.models.events import Event
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.contributions.models.principals import ContributionPrincipal
+from indico.modules.fulltextindexes.models.events import IndexedEvent
+
+
+def get_event_contributions_linked_to_user(user, from_dt=None, to_dt=None):
+    """Returns a dict with keys representing event_id and the values containing
+    data about the user rights for contributions within the event
+
+    :param user: A `User`
+    :param from_dt: The earliest event start time to look for
+    :param to_dt: The latest event start time to look for
+    """
+    event_date_filter = None
```

Comment:
This should be `True`, not `None` (it's also wrong in `get_events_created_by` and `get_events_managed_by`)